src/routes/safe/components/Apps/hooks/useIframeMessageHandler.ts

```diff
@@ -112,9 +113,10 @@ const useIframeMessageHandler = (
         data: SDKMessageToPayload[SDKMessageIds]
       }>,
     ) => {
-      if (message.origin === window.origin) {
+      if (message.origin === window.origin || message.origin === THIRD_PARTY_COOKIES_CHECK_URL) {
```

Comment:
Stylistic, but you can simplify this:
```suggestion
      if ([window.origin, THIRD_PARTY_COOKIES_CHECK_URL].includes(message.origin)) {
```